etlman/projects/tests/test_views/test_view_pipeline.py

```diff
@@ -119,6 +232,10 @@ def save_step2_data_in_session(self, client, pipeline, datainterface):
         }
         session.save()
 
+    def assert_session_cleared(self, client):
```

Comment:
I think this was meant for the other issue but didn't really work, right? No objections to committing it here anyways, just noting I think it still needs to be tested further.

Reply:
What's a good way to go about testing it? In what file should we write those tests? @tobiasmcnulty

Reply:
We decided to address this when we get work on #107